### PR TITLE
fix: Crash on Edge or Opera

### PR DESCRIPTION
**Pull request by parasharrajat**
Please review @Beamanator 

### Fixed Issues

$ https://github.com/Expensify/App/issues/5265

### Tests | QA Steps

 1. Open Microsoft Edge
 2. Navigate New Expensify
 3. Navigate to Profile or Preferences in settings

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### Edge
Screenshot 2021-09-16 20:40:43
### Opera
Screenshot 2021-09-17 17:33:59

### CODE REVIEW

**Comment by parasharrajat** [comment]
I am not sure why Marc is getting assigned to all of my PRs. :shield:

**Comment by Beamanator** [comment]
> I am not sure why Marc is getting assigned to all of my PRs. 🛡️

Thanks for noticing this, I am pretty sure I know why, a change to the PR template negatively affected how some internal auto-assigner was working 😅

**Comment by Beamanator** [comment]
@parasharrajat can you also test on Opera?

File: src/libs/getBrowser/index.web.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ function getBrowser() {
         temp = userAgent.match(/\b(OPR|Edge)\/(\d+)/);
```
Comment (Beamanator) [inline src/libs/getBrowser/index.web.js:18]: Looking at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent, I don't see a case `Edge` will be defined in the userAgent, especially for Opera. Should it be removed?

File: src/libs/getBrowser/index.web.js [hunk 2]
```diff
@@ -18,13 +18,13 @@ function getBrowser() {
...
 
         if (temp !== null) {
-            return temp.slice(1).replace('OPR', 'Opera');
+            return temp.slice(1).join(' ').replace('OPR', 'Opera');
```
Comment (Beamanator) [inline src/libs/getBrowser/index.web.js:21]: If I remember correctly, we decided we don't care about version number yet (this function is only for getting browser name). So I think we can just `return 'Opera';` here, and on line 27 below just `return 'Edge';` right?

**Review by Beamanator — CHANGES_REQUESTED** [review]
2 little comments - about Opera regex matching & returning browser w/out version # for opera and edge

**Comment by parasharrajat** [comment]
@Beamanator Changes done.

**Review by Beamanator — APPROVED** [review]
Nice, thanks man 👍

**Pull request merged**